Hymns/Common/WebView.swift

```diff
@@ -6,13 +6,18 @@ import WebKit
  */
 struct WebView: UIViewRepresentable {
 
-    let request: URLRequest
+    let url: URL?
 
     func makeUIView(context: Context) -> WKWebView {
         return WKWebView()
     }
 
     func updateUIView(_ uiView: WKWebView, context: Context) {
-        uiView.load(request)
+        guard let url = url else {
+            // TODO show error state
```

Comment:
Could you implement this error state?

Reply:
Not really sure what to do here... Doesn't it just return a blank now anyways?

Reply:
Yeah maybe just keep the TODO and we'll figure it out later